autopush/db.py

```diff
@@ -713,17 +793,22 @@ def drop_old_users(self, months_ago=2):
 
     @track_provisioned
     def update_message_month(self, uaid, month):
+        # type: (str, str) -> bool
         """Update the route tables current_message_month
 
         Note that we also update the last_connect at this point since webpush
-        users when connecting will always call this once that month.
+        users when connecting will always call this once that month. The
+        cuurrent_timestamp is also reset as a new month has no last read
+        timestamp.
 
         """
         conn = self.table.connection
         db_key = self.encode({"uaid": hasher(uaid)})
-        expr = "SET current_month=:curmonth, last_connect=:last_connect"
+        expr = ("SET current_month=:curmonth, last_connect=:last_connect"
+                ", current_timestamp=:timestamp")
         expr_values = self.encode({":curmonth": month,
-                                   ":last_connect": generate_last_connect()
+                                   ":last_connect": generate_last_connect(),
+                                   ":timestamp": 0,
```

Comment:
Oops, this isn't supposed to be here at all anymore, since the message record tracks the timestamp, not router. Leftover from first attempt at this.